resources/Materials/TestSuite/stdlib/noise/noise.mtlx

```diff
@@ -0,0 +1,86 @@
+<!--
+Basic noise tests:
+- cellnoise2d
+- cellnoise3d
+- fractal3d
+- noise2d
+- noise3d
+-->
+
+
+<?xml version="1.0"?>
+<materialx version="1.36">
+  <nodegraph name="cellnoise3d" type="" xpos="5.27769" ypos="14.6465">
+    <cellnoise3d name="cellnoise3d1" type="float" xpos="5.18398" ypos="4.94234">
+      <input name="position" type="vector3" value="0.0, 0.0, 0.0" output="out" nodename="multiply1" />
+    </cellnoise3d>
+    <output name="out" type="float" output="out" nodename="cellnoise3d1" />
+    <position name="position1" type="vector3" xpos="4.87103" ypos="7.95556">
+      <parameter name="space" type="string" value="object" />
```

Comment:
Can we add in a variant with world space since objects are now scaled by 10 there should be a difference. The code I saw generated for the marble example was always object space, but unsure if that's due to a generation error or you need to specify world space. I think it should always be world space but can discuss.